app/ids-rating/example.html

```diff
@@ -0,0 +1,8 @@
+<ids-layout-grid>
+    <ids-layout-grid-cell>
+        <ids-text><h2>Ratings</h2></ids-text>
```

Comment:
```suggestion
        <ids-text><h2>Ratings</h2></ids-text>
```

This should be ` <ids-text font-size="12" type="h1">Alerts</ids-text>` as the label/heading isnt styled right like other examples